doc/skill_db.txt

```diff
@@ -157,6 +157,31 @@ Sequence Map Form
 
 ---------------------------------------
 
+HitRate: Skill hit rate. Rate is 100 by default. If a skill is required to be learned, then the sequence map form is needed.
+
+Can be defined in scalar form or sequence map form:
+Scalar Form
+  HitRate: 120
+
+Sequence Map Form
+  HitRate:
+    - Level: 1
+      Count: 120
+    - Level: 2
+      Count: 140
+    - Level: 3
+      Count: 160
+    - Level: 4
+      Count: 180
+    - Level: 5
+      Count: 200
+    Skills: # Requires both NV_BASIC and SM_BASH to be learned.
```

Comment:
Maybe this description should be more explicit about the consequences when one of the skills is not learned (no hitrate bonus)